tests/runtime/filter_ecs.c

```
@@ -308,6 +403,10 @@ static void flb_test_ecs_filter_cluster_error()
     sleep(2);
     TEST_CHECK(expected.actual_records == expected.expected_records);
     filter_test_destroy(ctx);
+
+    /* unset env */
+    setenv("FLB_ECS_PLUGIN_UNDER_TEST", "", 1);
```

should a fail pattern be added here to ensure cluster name is not present?

---

Not essential